lib/usb/usb_process.js

```diff
@@ -5,6 +5,7 @@ var util = require('util');
 
 // Third Party
 var protocol = require('usb-daemon-parser');
+var debug = require('debug')('usb_process');
```

Comment:
The changes in this file aren't necessary for this fix but the debugging aid will be useful in the future.